### PR TITLE
:bug: use Message Tags in messenger

### PR DESCRIPTION
**Pull request by scottxxx666**

### CODE REVIEW

**Comment by liam-lai** [comment]
note: https://developers.facebook.com/docs/messenger-platform/reference/send-api/

**Comment by liam-lai** [comment]
maybe add API reference comment in code for people like me not familiar with fb API: 
https://developers.facebook.com/docs/messenger-platform/send-messages/message-tags/

File: messenger/request.go [hunk 1]
```diff
@@ -3,6 +3,8 @@ package messenger
 type Request struct {
 	Recipient `json:"recipient"`
 	Message   `json:"message"`
+	MessageType string `json:"message_type"`
+	Tag string `json:"tag"`
```
Comment (liam-lai) [inline messenger/request.go:7]: format issue, maybe install auto formatter at your editor?

File: messenger/messenger.go [hunk 2]
```diff
@@ -16,7 +16,7 @@ import (
 )
 
 const (
-	sendAPIURL    = "https://graph.facebook.com/v2.6/me/messages?access_token="
+	sendAPIURL    = "https://graph.facebook.com/v9.0/me/messages?access_token="
 	profileURL    = "https://graph.facebook.com/v2.6/me/messenger_profile?access_token="
```
Comment (liam-lai) [inline messenger/messenger.go:20]: do we need to update profile api to v9 as well?

Comment (dinos80152) [reply messenger/messenger.go:20]: I'm modify profile API in another PR

File: messenger/request.go (hunk 1 above)
Comment (scottxxx666) [reply messenger/request.go]: done

**Pull request merged**